osu.Game/Screens/Play/BeatmapMetadataDisplay.cs

```diff
@@ -159,6 +178,13 @@ private void load()
                 }
             };
 
+            starDifficulty = difficultyCache.GetBindableDifficulty(beatmap.BeatmapInfo);
+            starDifficulty.BindValueChanged(difficulty =>
+            {
+                if (difficulty.NewValue is StarDifficulty diff)
```

Comment:
i don't personally like null-checks done with `is`. they look redundant. would just null-check normally.

Reply:
well, I used it because I could shorten it from `difficulty.NewValue.Value` to `diff`, but I guess it's not a good pattern in general.